From 4f29f007daeac434b3c8dbc234a1d2e0c200ac1c Mon Sep 17 00:00:00 2001
From: yifang ma <yifangx.ma@intel.com>
Date: Tue, 3 Dec 2019 15:51:42 +0800
Subject: [PATCH] use private libdrm instead of external libdrm

Rename lib names to libxxx_orig, so that we use libdrm-intel instead of external libdrm,
and bunches of libdrm_pri renaming patches could be removed.

Tracked-On: OAM-88717
Signed-off-by: yifang ma <yifangx.ma@intel.com>
---
 Android.bp                        |  8 ++++----
 Android.sources.bp                |  2 +-
 amdgpu/Android.bp                 |  8 ++++----
 amdgpu/Android.sources.bp         |  2 +-
 etnaviv/Android.bp                |  8 ++++----
 etnaviv/Android.sources.bp        |  2 +-
 freedreno/Android.bp              |  8 ++++----
 freedreno/Android.sources.bp      |  2 +-
 intel/Android.bp                  |  8 ++++----
 intel/Android.sources.bp          |  2 +-
 libkms/Android.bp                 | 16 ++++++++--------
 libkms/Android.sources.bp         | 10 +++++-----
 nouveau/Android.bp                |  8 ++++----
 nouveau/Android.sources.bp        |  2 +-
 omap/Android.bp                   |  4 ++--
 radeon/Android.bp                 |  4 ++--
 tegra/Android.bp                  |  4 ++--
 tests/Android.bp                  |  2 +-
 tests/modetest/Android.bp         | 10 +++++-----
 tests/modetest/Android.sources.bp |  2 +-
 tests/proptest/Android.bp         |  6 +++---
 tests/util/Android.bp             | 12 ++++++------
 tests/util/Android.sources.bp     |  2 +-
 23 files changed, 66 insertions(+), 66 deletions(-)

diff --git a/Android.bp b/Android.bp
index 6fe434c..a075d8e 100644
--- a/Android.bp
+++ b/Android.bp
@@ -25,7 +25,7 @@ subdirs = ["*"]
 build = ["Android.sources.bp"]
 
 cc_defaults {
-    name: "libdrm_defaults",
+    name: "libdrm_defaults_orig",
     cflags: [
         // XXX: Consider moving these to config.h analogous to autoconf.
         "-DMAJOR_IN_SYSMACROS=1",
@@ -47,12 +47,12 @@ cc_defaults {
 
 // Library for the device
 cc_library {
-    name: "libdrm",
+    name: "libdrm_orig",
     recovery_available: true,
     vendor_available: true,
     defaults: [
-        "libdrm_defaults",
-        "libdrm_sources",
+        "libdrm_defaults_orig",
+        "libdrm_sources_orig",
     ],
 
     export_include_dirs: ["include/drm", "android"],
diff --git a/Android.sources.bp b/Android.sources.bp
index 73356dd..1e4f801 100644
--- a/Android.sources.bp
+++ b/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_sources",
+    name: "libdrm_sources_orig",
     srcs: [
         "xf86drm.c",
         "xf86drmHash.c",
diff --git a/amdgpu/Android.bp b/amdgpu/Android.bp
index 976f03e..e999eda 100644
--- a/amdgpu/Android.bp
+++ b/amdgpu/Android.bp
@@ -1,16 +1,16 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_amdgpu",
+    name: "libdrm_amdgpu_orig",
 
     cflags: [
 	"-DAMDGPU_ASIC_ID_TABLE=\"/vendor/etc/hwdata/amdgpu.ids\""
     ],
 
     defaults: [
-        "libdrm_defaults",
-        "libdrm_amdgpu_sources",
+        "libdrm_defaults_orig",
+        "libdrm_amdgpu_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/amdgpu/Android.sources.bp b/amdgpu/Android.sources.bp
index be85283..653ef89 100644
--- a/amdgpu/Android.sources.bp
+++ b/amdgpu/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_amdgpu_sources",
+    name: "libdrm_amdgpu_sources_orig",
     srcs: [
 	"amdgpu_asic_id.c",
         "amdgpu_bo.c",
diff --git a/etnaviv/Android.bp b/etnaviv/Android.bp
index 21deda9..fdddc65 100644
--- a/etnaviv/Android.bp
+++ b/etnaviv/Android.bp
@@ -1,11 +1,11 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_etnaviv",
+    name: "libdrm_etnaviv_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_etnaviv_sources",
+        "libdrm_defaults_orig",
+        "libdrm_etnaviv_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/etnaviv/Android.sources.bp b/etnaviv/Android.sources.bp
index aa82890..aaed155 100644
--- a/etnaviv/Android.sources.bp
+++ b/etnaviv/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_etnaviv_sources",
+    name: "libdrm_etnaviv_sources_orig",
     srcs: [
         "etnaviv_device.c",
         "etnaviv_gpu.c",
diff --git a/freedreno/Android.bp b/freedreno/Android.bp
index 9fca9b1..3d20772 100644
--- a/freedreno/Android.bp
+++ b/freedreno/Android.bp
@@ -1,11 +1,11 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_freedreno",
+    name: "libdrm_freedreno_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_freedreno_sources",
+        "libdrm_defaults_orig",
+        "libdrm_freedreno_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/freedreno/Android.sources.bp b/freedreno/Android.sources.bp
index 3c1ca31..808608c 100644
--- a/freedreno/Android.sources.bp
+++ b/freedreno/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_freedreno_sources",
+    name: "libdrm_freedreno_sources_orig",
     srcs: [
         "freedreno_device.c",
         "freedreno_pipe.c",
diff --git a/intel/Android.bp b/intel/Android.bp
index 22713ac..05aba7f 100644
--- a/intel/Android.bp
+++ b/intel/Android.bp
@@ -24,13 +24,13 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_intel",
+    name: "libdrm_intel_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_intel_sources",
+        "libdrm_defaults_orig",
+        "libdrm_intel_sources_orig",
     ],
     vendor: true,
 
     // Removed dependency to libpciaccess: not used on Android
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/intel/Android.sources.bp b/intel/Android.sources.bp
index 46e0328..e968e2e 100644
--- a/intel/Android.sources.bp
+++ b/intel/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_intel_sources",
+    name: "libdrm_intel_sources_orig",
     srcs: [
         "intel_bufmgr.c",
         "intel_bufmgr_fake.c",
diff --git a/libkms/Android.bp b/libkms/Android.bp
index b09dbf4..6521911 100644
--- a/libkms/Android.bp
+++ b/libkms/Android.bp
@@ -1,15 +1,15 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libkms",
+    name: "libkms_orig",
     defaults: [
-        "libdrm_defaults",
-        "libkms_sources",
-        "libkms_vmwgfx_sources",
-        "libkms_intel_sources",
-        "libkms_nouveau_sources",
-        "libkms_radeon_sources",
+        "libdrm_defaults_orig",
+        "libkms_sources_orig",
+        "libkms_vmwgfx_sources_orig",
+        "libkms_intel_sources_orig",
+        "libkms_nouveau_sources_orig",
+        "libkms_radeon_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/libkms/Android.sources.bp b/libkms/Android.sources.bp
index 5582f23..3addc57 100644
--- a/libkms/Android.sources.bp
+++ b/libkms/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libkms_sources",
+    name: "libkms_sources_orig",
     srcs: [
         "linux.c",
         "dumb.c",
@@ -10,28 +10,28 @@ cc_defaults {
 }
 
 cc_defaults {
-    name: "libkms_vmwgfx_sources",
+    name: "libkms_vmwgfx_sources_orig",
     srcs: [
         "vmwgfx.c",
     ],
 }
 
 cc_defaults {
-    name: "libkms_intel_sources",
+    name: "libkms_intel_sources_orig",
     srcs: [
         "intel.c",
     ],
 }
 
 cc_defaults {
-    name: "libkms_nouveau_sources",
+    name: "libkms_nouveau_sources_orig",
     srcs: [
         "nouveau.c",
     ],
 }
 
 cc_defaults {
-    name: "libkms_radeon_sources",
+    name: "libkms_radeon_sources_orig",
     srcs: [
         "radeon.c",
     ],
diff --git a/nouveau/Android.bp b/nouveau/Android.bp
index 12c37e3..a8a504d 100644
--- a/nouveau/Android.bp
+++ b/nouveau/Android.bp
@@ -1,11 +1,11 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_nouveau",
+    name: "libdrm_nouveau_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_nouveau_sources",
+        "libdrm_defaults_orig",
+        "libdrm_nouveau_sources_orig",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/nouveau/Android.sources.bp b/nouveau/Android.sources.bp
index 5ecdb53..72012e5 100644
--- a/nouveau/Android.sources.bp
+++ b/nouveau/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_nouveau_sources",
+    name: "libdrm_nouveau_sources_orig",
     srcs: [
         "nouveau.c",
         "pushbuf.c",
diff --git a/omap/Android.bp b/omap/Android.bp
index 05ca7d2..9c06cb4 100644
--- a/omap/Android.bp
+++ b/omap/Android.bp
@@ -1,12 +1,12 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_omap",
+    name: "libdrm_omap_orig",
     defaults: [
         "libdrm_defaults",
         "libdrm_omap_sources",
     ],
     vendor: true,
 
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/radeon/Android.bp b/radeon/Android.bp
index 9d0a09e..f2bd52e 100644
--- a/radeon/Android.bp
+++ b/radeon/Android.bp
@@ -1,11 +1,11 @@
 build = ["Android.sources.bp"]
 
 cc_library_shared {
-    name: "libdrm_radeon",
+    name: "libdrm_radeon_orig",
     defaults: [
         "libdrm_defaults",
         "libdrm_radeon_sources",
     ],
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 }
diff --git a/tegra/Android.bp b/tegra/Android.bp
index 33eaf6c..3839ebd 100644
--- a/tegra/Android.bp
+++ b/tegra/Android.bp
@@ -1,7 +1,7 @@
 cc_library_shared {
-    name: "libdrm_tegra",
+    name: "libdrm_tegra_orig",
     vendor: true,
-    shared_libs: ["libdrm"],
+    shared_libs: ["libdrm_orig"],
 
     srcs: ["tegra.c"],
 
diff --git a/tests/Android.bp b/tests/Android.bp
index cdc6c2c..696bcac 100644
--- a/tests/Android.bp
+++ b/tests/Android.bp
@@ -1,6 +1,6 @@
 subdirs = ["*"]
 
 cc_library_headers {
-    name: "libdrm_test_headers",
+    name: "libdrm_test_headers_orig",
     export_include_dirs: ["."],
 }
diff --git a/tests/modetest/Android.bp b/tests/modetest/Android.bp
index ca811fe..791f9a2 100644
--- a/tests/modetest/Android.bp
+++ b/tests/modetest/Android.bp
@@ -1,12 +1,12 @@
 build = ["Android.sources.bp"]
 
 cc_test {
-    name: "modetest",
+    name: "modetest_orig",
     defaults: [
-        "libdrm_defaults",
-        "modetest_sources",
+        "libdrm_defaults_orig",
+        "modetest_sources_orig",
     ],
 
-    shared_libs: ["libdrm"],
-    static_libs: ["libdrm_util"],
+    shared_libs: ["libdrm_orig"],
+    static_libs: ["libdrm_util_orig"],
 }
diff --git a/tests/modetest/Android.sources.bp b/tests/modetest/Android.sources.bp
index c6aca2e..e510bfe 100644
--- a/tests/modetest/Android.sources.bp
+++ b/tests/modetest/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "modetest_sources",
+    name: "modetest_sources_orig",
     srcs: [
         "buffers.c",
         "cursor.c",
diff --git a/tests/proptest/Android.bp b/tests/proptest/Android.bp
index 28c87e9..89affae 100644
--- a/tests/proptest/Android.bp
+++ b/tests/proptest/Android.bp
@@ -1,8 +1,8 @@
 cc_test {
-    name: "proptest",
+    name: "proptest_orig",
     defaults: ["libdrm_defaults"],
     srcs: ["proptest.c"],
 
-    shared_libs: ["libdrm"],
-    static_libs: ["libdrm_util"],
+    shared_libs: ["libdrm_orig"],
+    static_libs: ["libdrm_util_orig"],
 }
diff --git a/tests/util/Android.bp b/tests/util/Android.bp
index 36d1820..4aac168 100644
--- a/tests/util/Android.bp
+++ b/tests/util/Android.bp
@@ -24,12 +24,12 @@
 build = ["Android.sources.bp"]
 
 cc_library_static {
-    name: "libdrm_util",
+    name: "libdrm_util_orig",
     defaults: [
-        "libdrm_defaults",
-        "libdrm_util_sources",
+        "libdrm_defaults_orig",
+        "libdrm_util_sources_orig",
     ],
-    shared_libs: ["libdrm"],
-    header_libs: ["libdrm_test_headers"],
-    export_header_lib_headers: ["libdrm_test_headers"],
+    shared_libs: ["libdrm_orig"],
+    header_libs: ["libdrm_test_headers_orig"],
+    export_header_lib_headers: ["libdrm_test_headers_orig"],
 }
diff --git a/tests/util/Android.sources.bp b/tests/util/Android.sources.bp
index 529e112..c9a6f1a 100644
--- a/tests/util/Android.sources.bp
+++ b/tests/util/Android.sources.bp
@@ -1,7 +1,7 @@
 // Autogenerated with Android.sources.bp.mk
 
 cc_defaults {
-    name: "libdrm_util_sources",
+    name: "libdrm_util_sources_orig",
     srcs: [
         "format.c",
         "kms.c",
-- 
2.7.4

